app/controllers/katello/concerns/hosts_controller_extensions.rb

```diff
@@ -23,6 +26,35 @@ def puppet_environment_for_content_view
           cvpe = Katello::ContentViewPuppetEnvironment.where(:environment_id => environment, :content_view_version_id => version).first
           render :json => cvpe.nil? ? nil : {:name => cvpe.puppet_environment.name, :id => cvpe.puppet_environment.id}
         end
+
+        def content_hosts
```

Comment:
I think having a separate endpoint for 'content host' information is not ideal.  We've worked on unifying these concepts at the api and model layer and I think we should continue work towards that.  This would mean that the list of attributes for csv would need to be pluggable, but i think thats a better route and could be used for our extensions of hostgroup as well.